InnowiseSQL.sql

```diff
@@ -1 +1,406 @@
+	   ('Фаина','Стрелкова','Алексеевна','жен',2),
+	   ('Betty','Neumann','','жен',2),
+	   ('Hanna','Eder','','жен',1),
+	   ('Antonia','Klaus','','жен',3),
+	   ('Евгения','Аксёновa','','жен',1),
+	   ('Виктор','Калашников','Владимирович','муж',1),
+	   ('Майя','Щукинa','Владимировна','жен',4),
+	   ('Надежда','Вишнякова','Фёдоровна','жен',2),
+	   ('Эдуард','Кудряшов','Дмитриевич','муж',3),
+	   ('Евгения','Кашняр','Романовна','жен',4),
+	   ('Екатерина','Мясникова','Алексеевна','жен',1),
+	   ('Тарас','Красильников','Дмитриевич','муж',5),
+	   ('Спартак','Белоусов','Дмитриевич','муж',1);
+
+
+INSERT INTO Accounts(Bnk_ID,Balance, Cl_ID)
+VALUES (1,1566,1),
+	   (1,0,2),
+	   (1,0,3),
+	   (2,0,4),
+	   (2,78,5),
+	   (2,0,6),
+	   (3,3,7),
+	   (3,29,8),
+	   (3,0,9),
+	   (4,0,10),
+	   (4,19,11),
+	   (4,39,18),
+	   (5,533,22),
+	   (5,309,23),
+	   (5,399,24),
+	   (6,628,19),
+	   (6,570,20),
+	   (6,518,21),
+	   (7,796,25),
+	   (7,96,26),
+	   (7,394,27),
+	   (8,222,28),
+	   (8,638,29),
+	   (8,357,30),
+	   (9,300,12),
+	   (9,1031,13),
+	   (9,5000,14),
+	   (10,1166,15),
+	   (10,200,16),
+	   (10,562,17);
+
+INSERT INTO Cards(Balance,Ac_ID)
+VALUES (320,1),
+	   (200,1),
+	   (200,1),
+	   (200,1),
+	   (200,1),
+	   (0,4),
+	   (0,4),
+	   (0,4),
+	   (0,4),
+	   (200,24),
+	   (99,24),
+	   (531,13),
+	   (200,27),
+	   (94,27),
+	   (138,29),
+	   (100,23),
+	   (100,23),
+	   (100,23),
+	   (62,17),
+	   (0,10),
+	   (0,10),
+	   (0,14),
+	   (0,14),
+	   (0,7),
+	   (0,7),
+	   (0,7),
+	   (1,7),
+	   (1,7),
+	   (0,3),
+	   (40,25),
+	   (40,25),
+	   (40,25),
+	   (40,25),
+	   (40,25),
+	   (66,15),
+	   (100,15),
+	   (0,28),
+	   (0,9),
+	   (0,9),
+	   (0,16),
+	   (75,20),
+	   (122,22),
+	   (128,19),
+	   (100,19),
+	   (100,19),
+	   (100,19),
+	   (100,19);
+
+
+	--Query 1
+
+SELECT BankName, TownName
+FROM Banks
+INNER JOIN Filials ON Banks.BankID = Filials.Bnk_ID
+INNER JOIN Buildings ON Filials.FilialID = Buildings.Fil_ID
+INNER JOIN Towns ON Buildings.Twn_ID = Towns.TownID
+GROUP BY BankName, TownName
+HAVING TownName = 'Минск'; 
+
+	--Query 2
+
+SELECT BankName, CardID, AccountID, Name, Cards.Balance AS Card_Balance, Accounts.Balance AS Ac_Balance
+FROM Banks
+INNER JOIN Accounts ON Accounts.Bnk_ID = Banks.BankID 
+INNER JOIN Clients ON Accounts.Cl_ID = Clients.ClientID
+INNER JOIN Cards ON Cards.Ac_ID = Accounts.AccountID;
+
+
+	--Query 3
+
+SELECT AccountID, Accounts.Balance, SUM(Cards.Balance) AS Summa, Accounts.Balance - SUM(Cards.Balance) as dif
+FROM Accounts INNER JOIN Cards ON Ac_ID= AccountID
+GROUP BY AccountID, Accounts.Balance
+HAVING SUM(Cards.Balance) != Accounts.Balance
+
+	--Query 4a
+
+SELECT SocStatus.Name, COUNT(*) AS CardsAmount
+FROM Clients
+INNER JOIN Accounts ON ClientID = Cl_ID
+INNER JOIN SocStatus ON SocSt_ID = SocStatusID
+INNER JOIN Cards ON AccountID = Ac_ID
+GROUP BY SocStatus.Name;
+
+	--Query 4b  -
+
+	--Query 5
+
+CREATE PROCEDURE Calculating
```

Comment:
Change the name of procedure, because I don't understand what it should do, if I look only at name